blog/urls.py

```diff
@@ -0,0 +1,17 @@
+from django.urls import path
+
+from blog.views import (
+    index,
+    PostDetailView,
+    comment_create,
+    redirecttomain,
+)
+
+urlpatterns = [
+    path("", redirecttomain, name="redirect"),
```

Comment:
This redundant, you can just do
```suggestion
    path("", index, name="redirect"),
```